test/autotest.py

```diff
@@ -831,6 +831,7 @@ def saveResultsNMI():
 runTest("shared-memory2", 2, ["./test/shared-memory2"])
 S=DEFAULT_S
 
+runTest("cma",     2, ["./test/cma"])
```

Comment:
Can we change the indentation of the second argument, the "2",
to correspond to the indentation of the later lines?  (I know this is picky.)